ansible/group_vars/all.yml.example

```diff
@@ -34,4 +35,4 @@ notify_email_list: aweiteka@redhat.com
 # OpenShift client command options
 # --------------------------------
 # the token/clusterhost/project_name are set in host_vars for each environment
-oc: oc --token={{ token }} --server=https://{{ clusterhost }} -n {{ project_name }}
+oc: oc --insecure-skip-tls-verify --token={{ token }} --server=https://{{ clusterhost }} -n {{ project_name }}
```

Comment:
It's not clear to me why/if we should add `insecure-skip-tls-verify` by default... maybe add a comment instead mentioning that this is the place to add it if (absolutely) necessary?